druid_setup/single/environment/middlemanager.env

```diff
@@ -1,3 +1,5 @@
 druid.indexer.task.baseTaskDir=/data/output/druid/task
 
+druid_worker_capacity=6
```

Comment:
I'd say to remove this line. The choice of 6 was rather arbitrary and brazil specific